doc/examples/bundle_profiles.py

```diff
@@ -0,0 +1,183 @@
+
+transform = np.load("slr_transform.npy")
+
+"""
+
+In the next step, we need to make sure that all the streamlines in each bundles
+are oriented the same way. For example, for the CST, we want to make sure that
+all the bundles have their cortical termination at one end of the streamline.
+This is that when we later extract values from a volume, we won't have different
+streamlines going in opposite directions.
+
+To orient all the streamlines in each bundles, we will create standard
+streamlines, by finding the centroids of the left AF and CST bundle models.
+
+The advantage of using the model bundles is that we can use the same standard
+for different subjects, which means that we'll get roughly the same orientation
+"""
+import dipy.data as dpd
+from dipy.data.fetcher import get_two_hcp842_bundles
+model_af_l_file, model_cst_l_file = get_two_hcp842_bundles()
+
+model_af_l, hdr = load_trk(model_af_l_file)
+model_cst_l, hdr = load_trk(model_cst_l_file)
+
+
+from dipy.segment.metric import AveragePointwiseEuclideanMetric, ResampleFeature
+from dipy.segment.clustering import QuickBundles
+
+feature = ResampleFeature(nb_points=100)
+metric = AveragePointwiseEuclideanMetric(feature)
+
+"""
+
+Since we are going to include all of the streamlines in the single cluster
+from the streamlines, we set the threshold to `np.inf`. We pull out the centroid
+as the standard
+
+"""
+
+qb = QuickBundles(np.inf, metric=metric)
+
+cluster_cst_l = qb.cluster(model_cst_l)
+standard_cst_l = cluster_cst_l.centroids[0]
+
+cluster_af_l = qb.cluster(model_af_l)
+standard_af_l = cluster_af_l.centroids[0]
+
+"""
+We use the centroid streamline for each atlas bundle as the standard to orient
+all of the streamlines in each bundle from the individual subject. Here, the
+affine used is the one from the transform between the atlas and individual
+tractogram. This is so that the orienting is done relative to the space of the
+individual, and not relative to the atlas space.
+
+"""
+
+import dipy.tracking.streamline as dts
+
+oriented_cst_l = dts.orient_by_streamline(cst_l, standard_cst_l,
+                                          affine=transform)
+oriented_af_l = dts.orient_by_streamline(af_l, standard_af_l,
+                                         affine=transform)
+
+
+"""
+
+Read volumetric data from an image corresponding to this subject.
+
+For the purpose of this, we've extracted only the FA within the bundles in question,
+but in real use, this is where you would add the FA map of your subject.
+
+"""
+
+files, folder = dpd.fetch_bundle_fa_hcp()
+
+import nibabel as nib
+img = nib.load(op.join(folder, "hcp_bundle_fa.nii.gz"))
+fa = img.get_fdata()
+
+
+"""
+Calculate weights for each bundle:
+"""
+
+w_cst_l = dts.gaussian_weights(oriented_cst_l)
+w_af_l = dts.gaussian_weights(oriented_af_l)
+
+
+"""
+And then use the weights to calculate the bundle profiles for each bundle
+"""
+
+profile_cst_l = dts.bundle_profile(fa, oriented_cst_l, affine=img.affine)#,
```

Comment:
rename bundle_profile to afq_profile

Reply:
Please move afq_profile to dipy.segment.bundle module.

Reply:
The afq_profile could have done both the orientation and weights calculation. Not sure why you need independent functions for these.